config/example/config.toml

```diff
@@ -378,6 +379,29 @@ random_ai_lawset = true
 ################################################################
 
 
+[lighting_effects_configuration]
+# Settings that allow you to change light effects (bloom / exposure intensity)
+
+# Brightness of bloom effect independent of light_power
+glow_brightness_base = 0
+# Brightness of bloom effect that depends on light_power
+glow_brightness_power = -0.25
+# Contrast of bloom effect independent of light_power
+glow_contrast_base = 0
+# Contrast of bloom effect that depends on light_power
+glow_contrast_power = 0.5
+# Brightness of exposure effect independent of light_power
+exposure_brightness_base = 0.01
+# Brightness of exposure effect that depends on light_power
+exposure_brightness_power = 0
+# Contrast of exposure effect independent of light_power
+exposure_contrast_base = 9.5
+# Contrast of exposure effect that depends on light_power
+exposure_contrast_power = 0
+
```

Comment:
```suggestion


```
The nittiest of nitpicks. Purely a styling thing.

(I have comitted the config with the spacing and I wont need to re-review after this is handled)